package.json

```diff
@@ -20,7 +20,10 @@
   "devDependencies": {
```

Comment:
Got an error when I run `npm run test`
![image](https://user-images.githubusercontent.com/31897434/78050508-38871980-7374-11ea-8d1f-8e9c0a46dfde.png)

Hint: Try and add this -> `"@babel/preset-env": "^7.8.7"` to devDependencies

Reply:
thank you bellom we forgot that :smile: